news/4746.bugfix

```diff
@@ -0,0 +1 @@
+Redact username/password from log messages when installing packages from Git
```

Comment:
Update to say you're only redacting the password. Also, isn't `redact_password_from_url()` being called in non-Git cases, too, in some cases? I would just say "Redact password from the URL in some log messages."